### DISCUSSION TITLE
How to edit whoogle.env

### DISCUSSION THREAD

**Discussion by ProjectAI360** [benbusby/whoogle-search#828] [Q&A] [answered]
I've got a local install using docker and I'm a bit confused how to edit the whoogle.env file to add permanent settings. Where is this file?

**Comment by binitGajera** [comment, marked as answer]
This is the template file: [whoogle.template.env](https://github.com/benbusby/whoogle-search/blob/main/whoogle.template.env)

Since you're using docker, as the instructions mention on top of that template file, you can uncomment the `env_file` section from the [docker-compose.yml](https://github.com/benbusby/whoogle-search/blob/main/docker-compose.yml) file.

If you're using docker run then the alternative would be --env-file option, something like `docker run --env-file ./whoogle.env` where the value provided to the `--env-file` here and in the compose file would be the location of the environment file.
